Doc/library/decimal.rst

```diff
@@ -1480,17 +1480,17 @@ Constants
 The constants in this section are only relevant for the C module. They
 are also included in the pure Python version for compatibility.
 
-+---------------------+---------------------+-------------------------------+
-|                     |       32-bit        |            64-bit             |
-+=====================+=====================+===============================+
-| .. data:: MAX_PREC  | :const:`425000000`  | :const:`999999999999999999`   |
-+---------------------+---------------------+-------------------------------+
-| .. data:: MAX_EMAX  | :const:`425000000`  | :const:`999999999999999999`   |
-+---------------------+---------------------+-------------------------------+
-| .. data:: MIN_EMIN  | :const:`-425000000` | :const:`-999999999999999999`  |
-+---------------------+---------------------+-------------------------------+
-| .. data:: MIN_ETINY | :const:`-849999999` | :const:`-1999999999999999997` |
-+---------------------+---------------------+-------------------------------+
++---------------------+----------------------+--------------------------------+
+|                     |        32-bit        |             64-bit             |
++=====================+======================+================================+
+| .. data:: MAX_PREC  | :const:`!425000000`  | :const:`!999999999999999999`   |
++---------------------+----------------------+--------------------------------+
+| .. data:: MAX_EMAX  | :const:`!425000000`  | :const:`!999999999999999999`   |
++---------------------+----------------------+--------------------------------+
+| .. data:: MIN_EMIN  | :const:`!-425000000` | :const:`!-999999999999999999`  |
++---------------------+----------------------+--------------------------------+
+| .. data:: MIN_ETINY | :const:`!-849999999` | :const:`!-1999999999999999997` |
++---------------------+----------------------+--------------------------------+
```

Comment:
Using literals instead of `` :const:`!` `` would avoid having to modify all the extra lines here, which is a net savings in line diffs.